From 0025827224920f12bf33c345d32c9a44061b6b4f Mon Sep 17 00:00:00 2001
From: Edmunt Pienkowsky <roed@onet.eu>
Date: Wed, 28 Nov 2018 11:38:53 +0000
Subject: [PATCH 2/2]  Makefile - add ORANGEPI_ZERO_PLUS platform

Make it usable with DKMS.

Signed-off-by: Edmunt Pienkowsky <roed@onet.eu>

diff --git a/Makefile b/Makefile
index 9566b48..7c05ee1 100644
--- a/Makefile
+++ b/Makefile
@@ -44,6 +44,7 @@ CONFIG_RTL8723B = n
 CONFIG_RTL8814A = n
 CONFIG_RTL8723C = n
 CONFIG_RTL8188F = y
+CONFIG_RTL8189FS = m
 ######################### Interface ###########################
 CONFIG_USB_HCI = n
 CONFIG_PCI_HCI = n
@@ -137,6 +138,7 @@ CONFIG_PLATFORM_MOZART = n
 CONFIG_PLATFORM_RTK119X = n
 CONFIG_PLATFORM_NOVATEK_NT72668 = n
 CONFIG_PLATFORM_HISILICON = n
+CONFIG_PLATFORM_ORANGEPI_ZERO_PLUS = y
 ###############################################################
 
 CONFIG_DRVEXT_MODULE = n
@@ -1013,6 +1015,15 @@ CROSS_COMPILE := /home/android_sdk/Allwinner/a64/android-51/lichee/out/sun50iw1p
 KSRC :=/home/android_sdk/Allwinner/a64/android-51/lichee/linux-3.10/
 endif
 
+ifeq ($(CONFIG_PLATFORM_ORANGEPI_ZERO_PLUS), y)
+EXTRA_CFLAGS += -DCONFIG_LITTLE_ENDIAN
+EXTRA_CFLAGS += -DCONFIG_TRAFFIC_PROTECT
+EXTRA_CFLAGS += -DCONFIG_CONCURRENT_MODE
+EXTRA_CFLAGS += -DCONFIG_IOCTL_CFG80211 -DRTW_USE_CFG80211_STA_EVENT
+
+ARCH := arm64
+endif
+
 ifeq ($(CONFIG_PLATFORM_TI_AM3517), y)
 EXTRA_CFLAGS += -DCONFIG_LITTLE_ENDIAN -DCONFIG_PLATFORM_ANDROID -DCONFIG_PLATFORM_SHUTTLE
 CROSS_COMPILE := arm-eabi-
-- 
2.19.2

